tsl/test/expected/compress_bloom_sparse.out

```diff
@@ -845,3 +895,304 @@ drop table corner;
 drop table badtable;
 drop table byref;
 drop table arraybloom;
+-- Test configurable sparse indexes settings
```

Comment:
Can you move this to a separate file? They become unwieldy when too big. There's some overhead to creating a new db for a separate test file, but it's OK if the test is big enough.